public/index.pug

```diff
@@ -175,5 +175,7 @@ html(lang="en-US")
           title Twitter logo
           path(d=twitterIcon.path)
         span Share this on Twitter!
-
+      div(class="footer-repo")
+          p. 
+            #[a(title="github repository" href="https://github.com/simple-icons/simple-icons-website") Made with ❤️ on GitHub]
```

Comment:
Is `❤️` a [UTF-8](https://github.com/simple-icons/simple-icons-website/blob/6e919f786dd4413327113b2099a88524c7c99e96/public/index.pug#L4) character?

Reply:
It's a HTML entity now.